pkgs/top-level/all-packages.nix

```diff
@@ -28894,5 +28894,9 @@ in
 
   lc3tools = callPackage ../development/tools/lc3tools {};
 
+  wordlists = callPackage ../tools/security/wordlists/default.nix {
```

Comment:
also, you may want to change the name, it's already in use, which is why we have the 5k+ rebuilds

```
pkgs/development/libraries/cracklib/default.nix
3:, wordlists ? [ (fetchurl {
26:    ln -vs ${toString wordlists} dicts/
38:    license = licenses.lgpl21; # Different license for the wordlist: http://www.openwall.com/wordlists
```

Reply:
this is later used by pam, which why it's almost a full stdenv rebuild

```
pkgs/os-specific/linux/pam/default.nix
1:{ stdenv, buildPackages, fetchurl, fetchpatch, flex, cracklib, db4 }:
31:  buildInputs = [ cracklib db4 ];
```